dts/bindings/ethernet/wiznet,w5500.yaml

```diff
@@ -23,3 +23,9 @@ properties:
       The reset pin of W5500 is active low.
       If connected directly the MCU pin should be configured
       as active low.
+  ready-gpios:
```

Comment:
I'm wondering if this should be in a new device tree binding that inherits from this one. You're sorta imposing this requirement onto everyone even if they aren't using this module which is what I believe to be the more common case.

Reply:
@raveious an earlier revision of this PR had that, the maintainer did not agree, so it was removed

Reply:
I understand the differences between this module and underlying W5500 IC itself. Meaning, there is no ready-gpio from W5500. So to address that we can potentially introduce binding consuming the w5500 binding in include.